src/main/java/net/minestom/server/extensions/ExtensionManager.java

```diff
@@ -113,7 +114,7 @@ public void loadExtensions() {
                 attemptSingleLoad(discoveredExtension);
             } catch (Exception e) {
                 discoveredExtension.loadStatus = DiscoveredExtension.LoadStatus.LOAD_FAILED;
-                e.printStackTrace();
+                MinecraftServer.getExceptionManager().handleException(e);
                 LOGGER.error("Failed to load extension {}", discoveredExtension.getName());
                 LOGGER.error("Failed to load extension", e);
```

Comment:
This will print the stack trace twice (once with logger, once without). Probably the second log message should be gone and the exception handler should be called afterwards.

Reply:
Done.